addons/arsenal/functions/fnc_fillRightPanel.sqf

```diff
@@ -80,29 +80,13 @@ private _fnc_fillRightContainer = {
             _itemInfo = isClass (_cfgWeapons >> configName _configPath);
```

Comment:
I can't believe I didn't spot that. Looks like it didn't screw things up too badly though.

Reply:
Nuking that too, not needed anymore.